iii-directory/ui/styles.css

```
@@ -0,0 +1,490 @@
+  min-height: 100%;
+}
+[data-iii-ui="iii-directory"] .dir-ui-preview {
+  font-size: 13px;
+}
+[data-iii-ui="iii-directory"] .dir-ui-error {
+  padding: 8px 12px;
+  border-top: 1px solid var(--color-rule-2);
+  font-family: var(--font-mono, ui-monospace, monospace);
+  font-size: 12px;
+  color: var(--color-alert);
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)